WooCommerce/WooCommerceUITests/Tests/LoginTests.swift

```diff
@@ -12,45 +12,39 @@ final class LoginTests: XCTestCase {
         app.launch()
     }
 
-    // Login with Store Address and log out.
-    func skipped_test_site_address_login_logout() throws {
-        try skipTillSettingsFixed()
-
-        let prologue = try PrologueScreen().selectSiteAddress()
+    func test_site_address_login_logout() throws {
+        try PrologueScreen()
+            .selectSiteAddress()
             .proceedWith(siteUrl: TestCredentials.siteUrl)
             .proceedWith(email: TestCredentials.emailAddress)
             .proceedWith(password: TestCredentials.password)
-            .verifyEpilogueDisplays(displayName: TestCredentials.displayName, siteUrl: TestCredentials.siteUrl)
-            .continueWithSelectedSite()
 
-        // Log out
+        // Log out and verify
         try TabNavComponent()
             .goToMenuScreen()
+            .verifySelectedStoreDisplays(storeTitle: TestCredentials.storeName, storeURL: TestCredentials.siteUrl)
             .openSettingsPane()
-            .verifySelectedStoreDisplays(storeName: TestCredentials.storeName, siteUrl: TestCredentials.siteUrl)
             .logOut()
-
-        XCTAssert(prologue.isLoaded)
+            .verifyPrologueScreenLoaded()
     }
 
-    // Login with WordPress.com account and log out
-    func skipped_test_WordPress_login_logout() throws {
-        try skipTillSettingsFixed()
+    func test_WordPress_login_logout() throws {
 
-        let prologue = try PrologueScreen().selectContinueWithWordPress()
+        try PrologueScreen().selectContinueWithWordPress()
             .proceedWith(email: TestCredentials.emailAddress)
             .proceedWith(password: TestCredentials.password)
+
+        try LoginEpilogueScreen()
             .verifyEpilogueDisplays(displayName: TestCredentials.displayName, siteUrl: TestCredentials.siteUrl)
             .continueWithSelectedSite()
 
-        // Log out
+        // Log out and verify
```

Comment:
I'm not sure this comment is helpful, because we can clearly see some lines starting with .verify... and .logOut..., which makes it self-explanatory.

```suggestion
```

Reply:
Removed in 7e14865a197ae5090d509acd088985774bec135b